Decred Wallet/Features/History/TransactionHistoryViewController.swift

```diff
@@ -55,140 +59,135 @@ class TransactionHistoryViewController: UIViewController {
     
     override func viewDidAppear(_ animated: Bool) {
         if !AppDelegate.walletLoader.isSynced {
-            print(" wallet not synced on history")
+            print("wallet not synced on history")
             return
         }
-        self.visible = true
-        if (self.FromMenu){
-            refreshControl.showLoader(in: self.tableView)
-            loadTransactions()
-            FromMenu = true
-        }
+        self.refreshControl.showLoader(in: self.tableView)
+        loadTransactions()
     }
     
-    override func viewDidDisappear(_ animated: Bool) {
-        super.viewDidDisappear(animated)
-        self.visible = false
-        dismiss(animated: true, completion: nil)
+    func initFilterBtn() {
+        self.btnFilter.initMenu(self.filterMenu) { [weak self] index, value in
+            self?.applyTxFilter(currentFilter: DcrlibwalletTxFilterAll)
+        }
     }
     
-    @objc func handleRefresh(_ refreshControl: UIRefreshControl) {
-        self.loadTransactions()
+    func applyTxFilter(currentFilter: Int32) {
```

Comment:
`loadTransactions` should come before `applyTxFilter` based on the function call hierarchy.

Reply:
fixed in 9a8c24b